packages/headless/src/controllers/search-parameter-manager/headless-search-parameter-manager.test.ts

```diff
@@ -323,4 +325,49 @@ describe('search parameter manager', () => {
 
     expect(unavailableKeys).toEqual([]);
   });
+
+  describe('#synchronize', () => {
+    it('given partial search parameters, it dispatches #restoreSearchParameters with non-specified parameters set to their initial values', () => {
```

Comment:
Worth it in your opinion to do some kind of "mass check all parameters" test ?

Like a 
```
['q', 'enableQuerySyntax', 'aq', 'cq', ...].forEach(param=> it(`#restoreSearchParameters ${param}`, ()=> { ... })
```

Reply:
I don't think it's worthwhile because the test is only checking the destructuring. There are UTs for every parameter at the reducer level, checking that it is set in state correctly.